homeassistant/components/environment_canada/camera.py

```diff
@@ -78,17 +78,12 @@ def name(self):
         """Return the name of the camera."""
         if self.camera_name is not None:
             return self.camera_name
-        return " ".join([self.radar_object.station_name, "Radar"])
+        return "Environment Canada Radar"
 
     @property
     def device_state_attributes(self):
         """Return the state attributes of the device."""
-        attr = {
-            ATTR_ATTRIBUTION: CONF_ATTRIBUTION,
-            ATTR_LOCATION: self.radar_object.station_name,
-            ATTR_STATION: self.radar_object.station_code,
-            ATTR_UPDATED: self.timestamp,
-        }
+        attr = {ATTR_ATTRIBUTION: CONF_ATTRIBUTION, ATTR_UPDATED: self.timestamp}
```

Comment:
Changing state attributes is a breaking change. Please add a paragraph under the breaking change header in the PR description. Describe briefly what changed and what the user should do to cope with the breaking change.

Reply:
Thanks, I've updated the description to note the changes.